backend/node-register/apps.py

```diff
@@ -7,25 +7,28 @@
 from aiogrpc import RpcError
 
 logger = logging.getLogger(__name__)
+LEDGER = getattr(settings, 'LEDGER', None)
 
 
 class NodeRegisterConfig(AppConfig):
     name = 'node-register'
 
-    def ready(self):
-
-        # We try until success if dev mode, if it fails the backend will not start
-        # In production we let this app fails
+    def register_node(self, channel_name):
+        # We try until success, if it fails the backend will not start
         while True:
             try:
                 # args is set to empty because fabric-sdk-py doesn't allow None args for invoke operations
-                invoke_ledger(fcn='registerNode', args=[''], sync=True)
+                invoke_ledger(channel_name, fcn='registerNode', args=[''], sync=True)
             except RpcError as e:
                 if not settings.DEBUG:
                     raise
                 logger.exception(e)
                 time.sleep(5)
-                logger.info('Retry to register the node to the ledger')
+                logger.info(f'({channel_name}) Retry to register the node to the ledger')
             else:
-                logger.info('Node registered in the ledger')
+                logger.error(f'({channel_name}) Node registered in the ledger')
```

Comment:
Why do we switch from `info` to `error` here ?

Reply:
not sure. I reverted https://github.com/SubstraFoundation/substra-backend/pull/300/commits/30016d0922c7050db937678ce12ecf34be2a24e6